e2e/read-initial-options/async-config/jest.config.js

```diff
@@ -6,6 +6,6 @@
  */
 module.exports = async function () {
   return {
-    jestConfig: 'async-config',
+    jestConfig: 'jest-config',
```

Comment:
I meant to keep this as `async-config`

Reply:
Oh, I understood something else from your previous comment 😅
I think we should write separate tests in case of a custom string,
however, we can also do something like this in `test.each`
```
 if (directory === 'async-config'){
      expect(config).toEqual({jestConfig: 'async-config', rootDir});
    } else {
      expect(config).toEqual({jestConfig: filename, rootDir});
    }
```

Reply:
why doesn't a third argument on the list passed to `test.each` work?

Reply:
yes it works, done afb45092d0e9647516a451fcbcf65614352f9cf7
sorry for such confusion I'm still learning things....